### PR TITLE
enhancement: support customizing pluginConfig

### PR DESCRIPTION
**Pull request by lianghao208**
Fixes: #419 
/release-note-none

#### Does this PR introduce a user-facing change?
```
 NONE
```

### CODE REVIEW

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 1]
```diff
@@ -23,3 +23,33 @@ plugins:
   enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
   disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
 
+pluginConfig:
+  plugins:
+    - Coscheduling
+    - NodeResourceTopologyMatch
+    - NodeResourcesAllocatable
+    - LoadVariationRiskBalancing
+    - TargetLoadPacking
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:32]: Is it possible to remove this section? ideally, we should refrain from adding new "formats" of config if unnecessary.

In other words, does it technically work by incorporating the original pluginConfig:

```yaml
pluginConfig:
- name: foo
  args:
- name: bar
  args:
```

**Review by Huang-Wei — COMMENTED** [review]
Thanks for working on this.

File: manifests/install/charts/as-a-second-scheduler/values.yaml (hunk 1 above)
Comment (lianghao208) [reply manifests/install/charts/as-a-second-scheduler/values.yaml]: That's a good point, I've done some changes, PTAL.

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 2]
```diff
@@ -23,3 +23,27 @@ plugins:
...
+    scoringStrategy:
+      type: "LeastAllocated"
+- name: NodeResourcesAllocatable
+  args:
+    mode: Least
+    resources:
+      - name: cpu
+        weight: 1000000
+      - name: memory
+        weight: 1
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:42]: let's remove this part as well.

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 3]
```diff
@@ -23,3 +23,27 @@ plugins:
   enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
   disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
 
+pluginConfig:
+- name: Coscheduling
+  args:
+    permitWaitingTimeSeconds: 10
+    deniedPGExpirationTimeSeconds: 3
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:30]: this field is now removed in v1beta3.

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 4]
```diff
@@ -23,3 +23,27 @@ plugins:
   enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
   disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
 
+pluginConfig:
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:26]: Let's add some comments in this section:

```yaml
# Customize the enabled plugins' config.
# Refer to the "pluginConfig" section of manifests/<plugin>/scheduler-config.yaml.
# For example, for Coscheduling plugin, you want to customize the permit waiting timeout to 10 seconds:
- name: Coscheduling
  args:
    permitWaitingTimeSeconds: 10 # default is 60
# Or, customize the other plugins
# - name: NodeResourceTopologyMatch
#   args:
#     scoringStrategy:
#       type: MostAllocated # default is LeastAllocated
```

Comment (lianghao208) [reply manifests/install/charts/as-a-second-scheduler/values.yaml:29]: done.

Comment (lianghao208) [reply manifests/install/charts/as-a-second-scheduler/values.yaml:29]: @Huang-Wei I've updated this section, do I need to add more configs such as `manifests/<plugin>/scheduler-config-v1beta3.yaml` to support v1beta3?

Comment (Huang-Wei) [reply manifests/install/charts/as-a-second-scheduler/values.yaml:29]: We can leave that as a follow-up.

**Pull request merged**